cmd/publish/main.go

```diff
@@ -46,17 +46,22 @@ func New(ctx *PublishCommandContext) *cobra.Command {
 			}
 			return nil
 		},
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			outputFlag, _ := cmd.Flags().GetString("output")
+			if (outputFlag != "json") && (outputFlag != "yaml") && (outputFlag != "xml") {
+
+				messages := ctx.Messager.LoadVersionMessages(ctx.CliVersion)
+				defer func() {
```

Comment:
Maybe I'm wrong here, but I'm not sure we need the defer.  WDYT?

Reply:
No you're not wrong! It works the same without `defer` and we're not doing any work inside `PreRunE` function that require us make sure at the end of function we print messages using `defer`. I'm a noobie in golang 😅  not sure if this was written keeping some edge case in mind. 

I believe it is safe to remove `defer`. If you think same I can make code changes and submit the PR for review.

Thank You

Reply:
Yes, let's do it.

Reply:
Made changes.